test/blackbox/common/BlackboxTestsDiscovery.cpp

```diff
@@ -1365,6 +1365,115 @@ TEST_P(Discovery, AsymmeticIgnoreParticipantFlags)
     EXPECT_EQ(messages_on_port, allowed_messages_on_port);
 }
 
+//! Regression test for redmine issue 22506
+TEST_P(Discovery, single_unicast_pdp_response)
```

Comment:
It would be interesting to have a test using a user's flowcontroller limiting the bandwidth and two remote participants waiting for the PDP sample.

Reply:
Since there is currently no way to set a flow controller to the builtin writers, we will do this in a follow-up PR.